.github/workflows/bug_bounty.yml

```
@@ -0,0 +1,62 @@
+name: SalesforceCommerceCloud/pwa-kit/bug_bounty
+on:
+  # For testing - to be removed
+  push:
+    branches:
+      - 'mrt-test-ci-deploy'
+  # Run this job when a new Github release is published
+  release:
+    types: [released]
+
+jobs:
+  deploy-bug-bounty:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if latest release
+        id: release
+        run: |-
+          RELEASE_JSON=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/releases/latest")
+
+          LATEST_TAG_NAME=$(echo "$RELEASE_JSON" | jq -r '.tag_name')
+          RELEASE_EVENT_TAG_NAME=$(echo ${{ github.event.release.tag_name }})
+
+          echo "Latest tag: $LATEST_TAG_NAME"
+          echo "Release event tag: $RELEASE_EVENT_TAG_NAME"
+
+          if [ "$LATEST_TAG_NAME" == "$RELEASE_EVENT_TAG_NAME" ]; then
+            echo "We are releasing the latest. Continuing to deploy to bug bounty instance"
+            echo "IS_LATEST_RELEASE=true" >> $GITHUB_ENV
+          fi
+
+      - name: Checkout
+        if: env.IS_LATEST_RELEASE == 'true'
+        uses: actions/checkout@v4
+
```

Are we missing defining the Node version so we make sure we use one of the MRT supported versions?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/e130f4bde168da4781af47618fc3c2eec34f85cd/.github/workflows/test.yml#L170-L174

```suggestion
      - name: Setup Node
        if: env.IS_LATEST_RELEASE == 'true'
        uses: actions/setup-node@v4
        with:
          node-version: 22
          cache: 'npm'
```

---

I was thinking of relying on the pre-installed node so that we'd have less maintenance and not have to manually bump these node versions up but this is a fair point. We already have to update the node versions of other jobs manually so I'll add this

---

An simple idea could be to create a `.node-version` file at the root:
```
22
```

And update the Setup node step to:
```
- name: Setup node
  id: setup_node
  uses: actions/setup-node@v4
  with:
    node-version-file: '.node-version'
    cache: "npm"
```

It seem like people is using that approach:
https://github.com/search?q=node-version-file%3A+%27.node-version%27+path%3A*.yml&type=code